lxd/migrate_instance.go

```diff
@@ -729,20 +732,19 @@ func (s *migrationSourceWs) Do(state *state.State, migrateOp *operations.Operati
 	msg := migration.MigrationControl{}
 	err = s.recv(&msg)
 	if err != nil {
-		s.disconnect()
-		return err
+		return abort(err)
```

Comment:
Just for my understanding: the `abort` function doesn't call `s.disconnect()`. Will that cause issues?